frameworks/Scala/play2-scala/play2-scala-slick-netty.dockerfile

```diff
@@ -1,7 +1,9 @@
 FROM tfb/sbt:latest
 
-COPY ./play2-scala-slick ./
+ADD ./play2-scala-slick /play_app
+WORKDIR /play_app
 
 RUN sbt stage
 
-CMD target/universal/stage/bin/play2-scala-slick -Dplay.server.provider=play.core.server.NettyServerProvider
+CMD target/universal/stage/bin/play2-scala-slick
```

Comment:
just missing the `\` here at eol

Reply:
Thanks for debugging.  I admit I got impatient and didn't actually run the play2 tests locally.